src/status_im2/contexts/onboarding/generating_keys/view.cljs

```diff
@@ -54,7 +54,7 @@
                                                            "easing"   (:linear reanimated/easings))))))
   (reanimated/set-shared-value keys-saved-opacity
                                (reanimated/with-delay
-                                4600
+                                5100
```

Comment:
Please can you elaborate, how these values are created?
Also, should we store these values as a constant?

Reply:
yes magic numbers, we need to understand what's behind them, and if these numbers will work different on different devices

Reply:
@Parveshdhull I'm not certain which documentation they referred to when considering the time settings. I just tried to correct some miscalculations related to time that are causing overlaps.

sure will move them to constant values

Reply:
> @Parveshdhull I'm not certain which documentation they referred to when considering the time settings.

Probably @J-Son89 can help you out. https://github.com/status-im/status-mobile/pull/16415

We need to figure out how they are calculated, otherwise trial based solution might break on another device.

Reply:
@flexsurfer @Parveshdhull so these numbers introduced in this [PR](https://github.com/status-im/status-mobile/pull/16415), @J-Son89 could you please advice?

Reply:
@Parveshdhull i just moved these numbers to some proper constant values and waiting for Jamie to clarify